### DISCUSSION TITLE
do you need help

### DISCUSSION THREAD

**Discussion by renews** [naymspace/backpex#296] [General]
@Flo0807 Do you need help with the project? I see a lot PR's pending and some work required, I'm willing to help.
Also do you have a roadmap of feature you want to implement?
Cheers

**Comment by Flo0807** [comment]
Thanks for offering your help, we'd really appreciate your contribution! We are currently planning our roadmap and it would be great to hear your perspective on this. We will get back to you!

**Reply by pehbehbeh** [reply]
@renews We have moved the first batch from our internal issue tracker to GitHub.

- We have a first draft of our project roadmap available here: https://github.com/orgs/naymspace/projects/2
- Here you can see the things we are working on: https://github.com/orgs/naymspace/projects/1
- We also added a lot of our ideas to the discussion: https://github.com/naymspace/backpex/discussions/categories/ideas

I have also marked all issues with https://github.com/naymspace/backpex/labels/good-first-issue that are suitable for a first time contribution. Maybe there's something for you?
